packages/core/android/src/main/kotlin/com/datadog/reactnative/DdRumImplementation.kt

```diff
@@ -264,6 +264,15 @@ class DdRumImplementation(private val datadog: DatadogWrapper = DatadogSDKWrappe
         promise.resolve(null)
     }
 
+    /**
+     * Returns current session ID, or nil if unavailable.
```

Comment:
nit: `nil` is iOS-specific word

```suggestion
     * Returns current session ID, or null if unavailable.
```